composer.json

```diff
@@ -7,9 +7,91 @@
   "license": "GPL-2.0+",
   "minimum-stability": "dev",
   "prefer-stable": true,
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://packages.drupal.org/8"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "ckeditor-plugin/a11ychecker",
+        "version": "1.1.1",
+        "type": "drupal-library",
+        "dist": {
+          "url": "https://download.ckeditor.com/a11ychecker/releases/a11ychecker_1.1.1.zip",
+          "type": "zip"
+        },
+        "require": {
+          "composer/installers": "~1.0"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "ckeditor-plugin/balloonpanel",
+        "version": "4.14.0",
+        "type": "drupal-library",
+        "dist": {
+          "url": "https://download.ckeditor.com/balloonpanel/releases/balloonpanel_4.14.0.zip",
+          "type": "zip"
+        },
+        "require": {
+          "composer/installers": "~1.0"
+        }
```

Comment:
Anything list in the `repositories` section of dependant packages needs to be re-listed here so we can build and test with this. This is a non-root composer property.